src/attribute/Ownable.sol

```diff
@@ -20,8 +20,7 @@ abstract contract Ownable is IOwnable, Initializable {
 
     /// @notice Initializes the contract setting `msg.sender` as the initial owner
     function __Ownable__initialize() internal onlyInitializer {
-        if (owner() != address(0)) revert OwnableAlreadyInitializedError();
-        _updateOwner(_sender());
+        if (owner() == address(0)) _updateOwner(_sender());
```

Comment:
Could you explain a bit more about you had to remove this check? This was a pretty important safety check.

Reply:
The initializer used to be called only once, as version was hardcoded to 1.  As currently implemented in this branch, the initializer is called for each upgrade of the contract.  As such, this check would block any upgrade, because the owner would already be defined.

Reply:
Keep in mind, we still need to implement a facility to prevent reinitialization, which probably means ensuring only the proxy can reinitialize.  It makes sense to determine whether we want to move version (and name?) to the constructor first.

Reply:
This check is no longer removed.  Only the first deployment of an `Ownable`-derived contract should call `__Ownable__initialize()`.